app/controllers/api/v1x0/mixins/index_mixin.rb

```diff
@@ -24,20 +24,7 @@ def collection(base_query, pre_authorized: false)
         def rbac_scope(relation, pre_authorized: false)
           return relation if pre_authorized
 
-          access_scopes = pundit_user.access.scopes(relation.model.table_name, 'read')
-
-          if access_scopes.include?('admin')
-            relation
-          elsif access_scopes.include?('group')
-            ids = Catalog::RBAC::AccessControlEntries.new(pundit_user.group_uuids).ace_ids('read', relation.model)
-            relation.where(:id => ids)
-          elsif access_scopes.include?('user')
-            relation.by_owner
-          else
-            Rails.logger.error("Error in scope search for #{relation.model}")
-            Rails.logger.error("Scope does not include admin, group, or user. List of scopes: #{scopes}")
-            raise Catalog::NotAuthorized, "Not Authorized for #{relation.model}"
-          end
+          policy_scope(relation)
```

Comment:
👍

Reply:
@gmcculloug We started on that route by defining the resolve in the parent class and the sub classes could just inherit the method but since we only have Portfolio & Order as the main RBAC objects the PortfolioItem, OrderItem, ProgressMessages, Tags need some where to latch into. We will be redoing the PolicyObject so that we have a better model